manim/animation/transform_matching_parts.py

```diff
@@ -124,17 +124,20 @@ def __init__(
             fade_target.add(target_map[key])
         fade_target_copy = fade_target.copy()
 
-        if transform_mismatches:
-            if "replace_mobject_with_target_in_scene" not in kwargs:
-                kwargs["replace_mobject_with_target_in_scene"] = True
-            anims.append(Transform(fade_source, fade_target, **kwargs))
-        elif fade_transform_mismatches:
-            anims.append(FadeTransformPieces(fade_source, fade_target, **kwargs))
-        else:
-            anims.append(FadeOut(fade_source, target_position=fade_target, **kwargs))
-            anims.append(
-                FadeIn(fade_target_copy, target_position=fade_target, **kwargs),
-            )
+        if len(fade_source.submobjects) > 0 and len(fade_target.submobjects) > 0:
```

Comment:
I am not sure this is the best possible way of fixing things, I think it would be better if `FadeTransformPieces.ghost_to` (or `FadeTransform.ghost_to`) were made more robust such that the mobject is tested to check whether or not is has points before `.replace` is called.

Reply:
Done. Since it was unspecified what ghost_to should do if passed in an empty mobject, I arbitrarily decided the original mobject should just fade out in place. WDYT?